src/components/POLine/Item/util.js

```diff
@@ -28,10 +28,13 @@ export const checkInstanceIdField = (formValues, inventoryData) => {
 
     return contributor ? isEqual(contributor, el) : false;
   });
-  const isEqualProductIds = inventoryData.productIds.every(el => {
-    const productId = find(get(formValues, 'details.productIds', []), { 'productId': el.productId });
 
-    return productId ? isEqual(productId, el) : false;
+  const formProductIds = get(formValues, 'details.productIds', []);
+
+  const isEqualProductIds = formProductIds.every(item => {
+    const productId = find(inventoryData.productIds, { 'productId': item.productId });
+
+    return productId ? isEqual(productId, item) : false;
```

Comment:
does it work if user adds product?

Reply:
When we add new productId to list we remove the Instance UUID. Also we remove the Instance UUID if change some product.

Reply:
We still attached to instance uuid only if removed.

Reply:
but earlier it worked - instance uuid was attached if user adds product id and type, as I can see

Reply:
as I understand it, we are changing this behavior. Look in the ticket please https://issues.folio.org/browse/UIOR-374

Reply:
I looked into the ticket and found there info only about `Allow deletion`. I think the rest behavior should stay the same

Reply:
yes, scenario 2 contains that changed condition